### ISSUE TITLE
Deployment details in the Azure portal does not show the deployed resources

### ISSUE THREAD

**Issue by danstis** [microsoftgraph/msgraph-bicep-types#35]
When deploying a new application using the Preview provider, a deployment record is created at the deployment scope (Resource Group or Subscription) however it contains no details of what was deployed.

For example:
Correlation ID: f2d5e313-ff86-4715-acf6-82071fdde4da
![image](https://github.com/microsoftgraph/msgraph-bicep-types/assets/2137928/48ddc731-c80e-4e11-8bda-4085c1145e0a)

The deployment details when expanded, shows the loading animation, but never displays any details.

**Comment by Wopienkaatwork** [comment]
Hi, I just tried the new preview features as well but was not able to deploy graph resources via the portal : 
```
extension microsoftGraph


resource group 'Microsoft.Graph/groups@v1.0'= {
  displayName: 'abc'
  mailEnabled: false
  mailNickname: 'abc'
  securityEnabled: true
  uniqueName: 'abc'
}
```
I took the json and tried to deploy via the portal as a custom deployment and the only message I got was this: 
![image](https://github.com/user-attachments/assets/6e73312a-04e1-4152-9825-2096bcb3bc65)
Am I missing something for the portal deployment?

**Comment by dkershaw10** [comment]
Yes this is expected - please visit the [known issues topic](https://learn.microsoft.com/graph/templates/known-issues-graph-bicep) and the [limitations topic](https://learn.microsoft.com/graph/templates/limitations).
There are two issues:
1. Only certain clients are currently allowed to perform deployments (for Graph resources) - see [issue item](https://learn.microsoft.com/graph/templates/known-issues-graph-bicep#deployment-error-this-application-0-isnt-authorized-to-call-microsoft-graph-using-a-bicep-template)
2. Deployment for extensible resources are not supported in the Azure Portal - see [limitation item](https://learn.microsoft.com/graph/templates/limitations#unsupported-deployment-features)

@Wopienkaatwork Now for the issue, we **could** look into adding the Azure Portal to the set of clients allowed to deploy Graph resources. If this is something important for you, please create a new issue and select "Feature request" and fill in the template.

**Comment by dkershaw10** [comment]
@danstis while the Azure Portal doesn't show deployed resources (and that'll be something we work on in the future), you can now at least see a resource's request and response information, if you deploy using **Debug mode**. I realize that this isn't great if your template is deploying a **lot** of Graph resources.

<img width="1043" height="317" alt="Image" src="https://github.com/user-attachments/assets/d9a64690-160d-4fe5-aeb2-927167f9a9f5" />

Shows that the Resource and Status are empty (and nothing to indicate the type).  But if you click on the **Operation details** link you can now see the template request and the Microsoft Graph response:

<img width="1402" height="1187" alt="Image" src="https://github.com/user-attachments/assets/4faad96f-56e6-4929-b456-8e334f90123c" />

Hope this helps (a bit at least).
